calibr8/tests.py

```diff
@@ -774,6 +774,22 @@ def test_infer_independent_not_implemented(self):
 
 
 class TestModelFunctions:
+    def test_exponential(self):
+        x = numpy.array([1.,2.,4.])
+        theta = [2,4]
+        expected = 2*(1-numpy.exp(-4*x))
+        true = calibr8.exponential(x, theta)
+        assert (numpy.array_equal(true, expected))
```

Comment:
```suggestion
        numpy.testing.assert_array_equal(expected, true)
```
gives more informative error messages